Add Hai Reverse Thrust Oufits (#7287)
endless-sky/endless-sky: data/hai/hai outfits.txt

	"flare sound" "atomic large"
	description "These thrusters are the largest of the Hai atomic thrusters, and one of the most powerful thrusters in known space. Beat out by only the largest human atomic thrusters, these massive engines provide enough thrust to push around even the largest of ships as if they were fighters."

outfit `"Basrem" Reverse Thruster`
	category "Engines"
	"cost" 150000
	thumbnail "outfit/tiny atomic reverse hai"
	"mass" 18
	"outfit space" -18
	"weapon capacity" -18
	"reverse thrust" 12.5
	"reverse thrusting energy" 1.4
	"reverse thrusting heat" 2.8
	"reverse flare sprite" "effect/atomic flare/tiny"
		"frame rate" 14
	"reverse flare sound" "atomic tiny"

> 	description "This tiny reverse thruster allows even the smallest of Hai ships to mount them for additional maneuverability beyond normal steering. Many young Hai find these an invaluable investment when first setting out on their own."

Reviewer: ```suggestion
	description "This tiny reverse thruster allows even the smallest of Hai ships to mount it for additional maneuverability beyond normal steering. Many young Hai find these an invaluable investment when first setting out on their own."
```

> 	description "This tiny reverse thruster allows even the smallest of Hai ships to mount them for additional maneuverability beyond normal steering. Many young Hai find these an invaluable investment when first setting out on their own."

Reviewer: ```suggestion
	description "These tiny reverse thrusters allow even the smallest of Hai ships to mount them for additional maneuverability beyond normal steering. Many young Hai find these an invaluable investment when first setting out on their own."
```

> 	description "This tiny reverse thruster allows even the smallest of Hai ships to mount them for additional maneuverability beyond normal steering. Many young Hai find these an invaluable investment when first setting out on their own."

Reviewer: ```suggestion
	description "This tiny reverse thruster allows even the smallest of Hai ships additional maneuverability beyond normal steering. Many young Hai find these an invaluable investment when first setting out on their own."
```

First two depending on which way you want to go on number agreement there (but plural would be most consistent with other entries), but honestly it's a bit awkward either way; third just skips the problem entirely and is far smoother.

You could also trim the phrase and still switch it to plural to be consistent with other entries, if you prefer that.

outfit `"Benga" Reverse Thruster`
	category "Engines"
	"cost" 300000
	thumbnail "outfit/small atomic reverse hai"
	"mass" 28
	"outfit space" -28
	"weapon capacity" -28
	"reverse thrust" 22.4
	"reverse thrusting energy" 2.5
	"reverse thrusting heat" 4.8
	"reverse flare sprite" "effect/atomic flare/tiny"
		"frame rate" 14
	"flare sound" "atomic tiny"

> 	description "Many Hai freighter pilots install reverse thrusters such as these to supplement the often poor steering on many larger Hai vessels. Considering them to be well worth the steep cost."

Reviewer: ```suggestion
	description "Considering them to be well worth the steep cost, many Hai freighter pilots install reverse thrusters such as these to supplement the often poor steering on larger Hai vessels."
```

That's just a big ol hanging fragment.

Repetitive "many" can go, too.

> 	description "Many Hai freighter pilots install reverse thrusters such as these to supplement the often poor steering on many larger Hai vessels. Considering them to be well worth the steep cost."

Reviewer 2: ```suggestion
	description "Many Hai freighter pilots install reverse thrusters such as these to supplement the often poor steering on many larger Hai vessels, considering them to be well worth the steep cost."
```
Still doesn't feel 'smooth' to me, but better than a separate sentence

outfit `"Biroo" Reverse Thruster`
	category "Engines"
	"cost" 610000
	thumbnail "outfit/medium atomic reverse hai"
	"mass" 44
	"outfit space" -44
	"weapon capacity" -44
	"reverse thrust" 39.4
	"reverse thrusting energy" 4
	"reverse thrusting heat" 8.2
	"reverse flare sprite" "effect/atomic flare/small"
		"frame rate" 13
	"reverse flare sound" "atomic small"

> 	description "This is the largest reverse thruster the Hai offer, preferring to focus on additional steering when more maneuverability is needed. Despite that, these reverse thrusters boast an impressive amount of power for their size, allowing even large ships to quickly come to a stop when needed."

Reviewer: ```suggestion
	description "With pilots generally preferring to focus on steering when actual agility is needed, these are the largest reverse thrusters the Hai offer. They still boast an impressive amount of power for their size, which allows even the largest Hai ships to quickly come to a stop when needed."
```
Another bad fragment. Simplest fix  would be to just add the mention of pilots, but rearranging too gets much smoother flow.

Went ahead and made this one plural as well.
